content/doc/book/pipeline/jenkinsfile.adoc

```
@@ -652,64 +541,34 @@ pipeline {
 
 ==== For other credential types
 
-If you need to set credentials in a Pipeline for anything other than secret
-text, usernames and passwords, or secret files
-(<<#for-secret-text-usernames-and-passwords-and-secret-files,above>>) - i.e SSH
-keys or certificates, then use Jenkins' *Snippet Generator* feature, which you
-can access through Jenkins' classic UI.
+If you need to set credentials in a Pipeline for anything other than secret text, usernames and passwords, or secret files(<<#for-secret-text-usernames-and-passwords-and-secret-files,above>>) - i.e SSH keys or certificates, then use Jenkins' *Snippet Generator* feature, which you can access through Jenkins' classic UI.
```

```suggestion
If you need to set credentials in a Pipeline for anything other than secret text, usernames and passwords, or <<#for-secret-text-usernames-and-passwords-and-secret-files,secret files>> like SSH keys or certificates, use Jenkins' *Snippet Generator* feature, which you can access through Jenkins' classic UI.
```
Suggestion to adjust link to not use [`above`](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/a/above) and [replace `i.e` usage ](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/i/ie).